app/markets/ActivityChart.tsx

```diff
@@ -0,0 +1,192 @@
+      tradesBlockNumbers = Array.from(new Set(tradesBlockNumbers));
+
+      if (!tradesBlockNumbers) return;
+
+      const queryOutcomeTokenAmounts = `query outcomeTokenAmountsByBlockNumber($id: ID!) {
+          ${tradesBlockNumbers.map(
+            timestamp => `_${timestamp}: fixedProductMarketMaker(id: $id, block: { number: ${timestamp} }) {
+              outcomeTokenAmounts
+            }`
+          )}
+        }`;
+
+      const outcomeTokenAmounts = await request<OutcomesDataType>(
+        OMEN_SUBGRAPH_URL,
+        queryOutcomeTokenAmounts,
+        {
+          id,
+        }
+      );
+
+      return Object.values(outcomeTokenAmounts).map((value, index) => {
+        const outcomes = Object.assign({ 0: '', 1: '' }, value.outcomeTokenAmounts);
+        return { time: tradesTimestamp[index], ...outcomes };
+      });
+    },
+    enabled: !!trades,
+  });
+
+  if (!data || !market)
+    return <div className="h-full w-full animate-pulse rounded-8 bg-outline-low-em" />;
+
+  const TooltipContent = (chartData: DefaultTooltipContentProps<number, string>) => {
+    const { payload } = chartData;
+
+    if (!payload || !payload[0] || !payload[1]) return;
+
+    const total = payload.reduce(
+      (result, entry) => (entry.value ? result + +entry.value : 0),
+      0
+    );
+    const tradeTime = format(fromUnixTime(+payload[0].payload.time), 'HH:mm dd/MM/yy');
+
+    return (
+      <div className="rounded-12 bg-surface-surface-0 p-3 shadow-4">
+        <p>Bets: {data.length}</p>
+        <ul>
+          {payload
+            .sort((a, b) => {
+              if (!a.dataKey || !b.dataKey) return 0;
+              return +a.dataKey - +b.dataKey;
+            })
+            .map((entry, index) => (
+              <li
+                key={index}
+                className={cx({
+                  'text-text-danger-em': entry.dataKey === '0',
+                  'text-text-success-em': entry.dataKey === '1',
+                })}
+              >
+                {`${entry.name}: ${entry.value ? getPercent(entry.value, total) : '-'}`}
+              </li>
+            ))}
+        </ul>
+        <p className="text-text-low-em">{tradeTime}</p>
+      </div>
+    );
+  };
+
+  const lastDataPoint = data[data.length - 1];
+  const total = +lastDataPoint[0] + +lastDataPoint[1];
+
+  const marketModel = new Market(market);
+
+  return (
+    <ResponsiveContainer width="100%" height="100%" className="border-radius-svg">
+      <AreaChart
+        width={500}
+        height={400}
+        data={data}
+        stackOffset="expand"
+        margin={{
+          left: 0,
+          bottom: 0,
+        }}
+      >
+        <Tooltip content={TooltipContent} />
+        <Area
+          type="monotone"
+          dataKey="1"
+          name={marketModel.outcomes[0].symbol}
+          stackId="1"
+          fill={resolvedTheme === 'light' ? '#AFFAD0FF' : '#134228'}
```

Comment:
Suggestion:
`const isLightheme = resolvedTheme === 'light'`

```suggestion
          fill={isLightheme ?  '#AFFAD0FF' : '#134228'}
```